client/my-sites/media-library/list-no-results.jsx

```diff
@@ -19,54 +20,72 @@ class MediaLibraryListNoResults extends React.Component {
 	static propTypes = {
 		filter: PropTypes.string,
 		search: PropTypes.string,
+		queryFilters: PropTypes.object,
 	};
 
 	static defaultProps = {
 		search: '',
+		queryFilters: null,
 	};
 
+	hasActiveQueryFilters() {
+		return this.props.queryFilters && ! isEmpty( this.props.queryFilters.dateRange );
+	}
+
 	getLabel = () => {
 		let label;
+		let searchOrQueryFilter;
```

Comment:
This kind of translation generally doesn't work well for some languages as they may need to change the whole sentence structure, which isn't possible when two separately translated phrases are joined in such a way.

I understand there's a lot of variants here and it may be possible to reduce this by changing the message. 

For example, things like:

`No images match %s.` and separately `the search and/or filters you applied`

Would need to be a single phrase:

`No images match the search and/or filters you applied`

Reply:
I was wondering about that. Now you say it and in the clear light of day I can see this wouldn't work.

Feels like it will need a wholesale refactor to account for the variations. An alternative is providing a more generic message.

I'm going to have to think about this one. As you say it might need to be its own PR.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/25762/commits/0bca5735323b0e606e34dd9bce0060bfe39057d1